.github/workflows/README.md

```diff
@@ -2,52 +2,123 @@
+This workflow is triggered on push events to the `dev`, `staging`, and `production` branches, and specific file modifications.
+
+#### Environment Variables
+
+- `PRODUCT` 📦: Represents the name of the product ("apim").
+- `ENVIRONMENT` 🌍: Represents the name of the environment, which is retrieved from the GitHub ref name.
+- `TIMEZONE` 🕒: Specifies the timezone as "Europe/London."
+- `FROM` 📁: Represents a base artifact, with the value "latest."
 
-### Bash
-This is a Bash script for a deployment strategy for a project. It prompts the user with a list of deployment destinations and asks for input to select the destination. The script sets the destination and the branch for the deployment based on the user's input. If the user selects ACR Purge, the script runs the az acr run command to purge the ACR repository based on specified filter and ago duration. After setting the destination and branch, the script displays the latest push commit for the selected branch, creates a new branch based on the destination, and pushes the changes to that branch. Then the script cleans up the local branches and notifies the user that the deployment has been initiated. The script version and author information are also provided at the end of the script.
+#### Jobs
 
-## SCA
-This is a GitHub Action YAML file that sets up environment variables and performs various source code analysis (SCA) tasks on the ESTORE-API project when a pull request is made to the main branch. The code quality SCA is performed by Codacy and the licensing SCA is performed by Fossa.
-The first job, "setup," sets the environment variables and outputs them for use in later jobs. The next two jobs, "codacy" and "license," run on the "ubuntu-latest" operating system and are triggered by the "setup" job's completion.
-The "codacy" job uses the Codacy analysis CLI action to perform SCA for code quality. Similarly, the "license" job uses the Fossa action to perform SCA for licensing.
-Secrets, such as the Fossa API key, are assumed to be set up beforehand in the repository's secrets.
+##### 1. Setup 🔧
 
-## TEST
-This is a GitHub Actions workflow that performs QA and then runs three different types of tests on a pull request on the `main` branch:
+- This job sets up deployment variables.
+- It runs on a self-hosted runner with the "APIM" and "deployment" labels.
+- Outputs:
+  - `product`: Contains the value of the `PRODUCT` environment variable.
+  - `environment`: Contains the value of the `ENVIRONMENT` environment variable.
+  - `timezone`: Contains the value of the `TIMEZONE` environment variable.
+- Steps:
+  - Environment 🧪: Displays the environment set to the `ENVIRONMENT` value.
+  - Timezone 🌐: Displays the timezone set to the `TIMEZONE` value.
 
-* Unit tests
-* API tests
-* End to end (E2E) tests.
+##### 2. MDM 📦️
 
-The workflow consists of four jobs:
+- This job represents the deployment of the MDM (Master Data Management) micro-service.
```

Comment:
eStore?

Reply:
Have replaced all MDM instances to ESTORE.